.github/workflows/ui_test.yml

```
@@ -0,0 +1,58 @@
+name: testing workflow
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: ikalnytskyi/action-setup-postgres@v5
+      - name: Postgres setup
+        run: |
+          createdb --owner $PGUSER $PGDATABASE
+          psql -c "ALTER USER $PGUSER WITH PASSWORD '$PGPASSWORD'"
+        env:
+          PGHOST: localhost
+          PGDATABASE: evershop
+          PGUSER: postgres
+          PGPASSWORD: postgres
```

```suggestion
        env:
          PGHOST: localhost
          PGDATABASE: evershop
          PGUSER: postgres
          PGPASSWORD: postgres
```
 same as this comment https://github.com/ishabaral/evershop/pull/5#discussion_r1566842138